docs/pages/technical/fhir-rules.rst

```diff
@@ -54,6 +55,14 @@ In order for the Nuts components to use the consent records for validation, addi
 
 Each complex requirement is explained in sub sections.
 
+Meta
+....
+
+The :code:`meta` field is used to track changes to the consent and might indicate there are previous versions. Only the latest consent proof will be referenced form the active document.
```

Comment:
Meta feels like a hack. The versionId and lastUpdated are also not defined FHIR fields right? If we use a DocumentReference instead of an attachment we can probably put this info in there.

Reply:
They are official fields for every resource. Updating it in the doc reference is a bit weird since the entire document probably changed.